plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -1156,9 +1156,15 @@ public boolean configure(final String name, final Map<String, Object> params) th
         storageProcessor.configure(name, params);
         storageHandler = new StorageSubsystemCommandHandlerBase(storageProcessor);
 
-        IscsiStorageCleanupMonitor isciCleanupMonitor = new IscsiStorageCleanupMonitor();
-        final Thread cleanupMonitor = new Thread(isciCleanupMonitor);
-        cleanupMonitor.start();
+        boolean _iscsiCleanUpEnabled = (boolean)params.get("iscsi.session.cleanup.enabled");
+
+        if (_iscsiCleanUpEnabled) {
+            IscsiStorageCleanupMonitor isciCleanupMonitor = new IscsiStorageCleanupMonitor();
+            final Thread cleanupMonitor = new Thread(isciCleanupMonitor);
+            cleanupMonitor.start();
+        } else {
+            s_logger.info("iscsi session clean up is disabled")
```

Comment:
I believe you have forgotten the ';' on here.

Reply:
fixed